src/sdam/common.ts

```diff
@@ -58,7 +58,7 @@ export function drainTimerQueue(queue: TimerQueue): void {
 /** @public */
 export interface ClusterTime {
   clusterTime: Timestamp;
-  signature: {
+  signature?: {
```

Comment:
Public API was incorrect

Reply:
Seems fine as a fix but in a separate PR.  One line fix PR maybe?